content/_includes/birds.macros.njk

```diff
@@ -101,3 +103,48 @@ params:
     <span {{ utility.attr_if('class', class) }}>{{ name | capitalize }}</span>
   {%- endif -%}
 {%- endmacro %}
+
+{% macro card(
+  bird,
+  birds=none,
+  level=3,
+  link=true,
+  bio=false,
+  class=''
```

Comment:
It looks like we're not using any of these in practice -- do we want to remove some (e.g. `bio`), or do we think we'll use them?

Reply:
Good catch. I built this out for using on the workshop page, and then it didn't look great, so I abandoned it. Bio is removed. I think the others still help make it re-usable if we ever want.